deploy-scripts/deploy_release.sh

```diff
@@ -13,7 +13,7 @@ elif [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   fail "Failed release deployment: was pull request."
 elif [ "$TRAVIS_BRANCH" != "$BRANCH" ]; then
   fail "Failed release deployment: wrong branch. Expected '$BRANCH' but was '$TRAVIS_BRANCH'."
-elif ! [ "$VERSION_NAME" =~ $VERSION_PATTERN ]; then
+elif ! [[ "$VERSION_NAME" =~ $VERSION_PATTERN ]]; then
```

Comment:
I swear to god I thought I fixed that. Maybe in another file.

Reply:
It was, in deploy_snapshots 😓